src/sardana/pool/poolutil.py

```diff
@@ -56,8 +66,15 @@ def get_device(self, *args, **kwargs):
         return dev
 
     get_motor = get_phy_motor = get_pseudo_motor = get_motor_group = \
-        get_exp_channel = get_ct_channel = get_zerod_channel = get_oned_channel = \
-        get_twod_channel = get_pseudo_counter_channel = get_measurement_group = \
-        get_com_channel = get_ioregister = get_device
+        get_exp_channel = get_ct_channel = get_zerod_channel = \
+        get_oned_channel = get_twod_channel = get_pseudo_counter_channel = \
+        get_measurement_group = get_com_channel = get_ioregister = get_device
```

Comment:
The use of backslash for breaking lines is discouraged by PEP8 . But using parentheses in this case is quite ugly (see below what, e.g. `black` would do), so I am not changing it:

```python
    get_motor = (
        get_phy_motor
    ) = (
        get_pseudo_motor
    ) = (
        get_motor_group
    ) = (
        get_exp_channel
    ) = (
        get_ct_channel
    ) = (
        get_zerod_channel
    ) = (
        get_oned_channel
    ) = (
        get_twod_channel
    ) = (
        get_pseudo_counter_channel
    ) = get_measurement_group = get_com_channel = get_ioregister = get_device
```